exporters/prometheus/CMakeLists.txt

```diff
@@ -17,23 +17,23 @@ if(NOT TARGET prometheus-cpp::core)
   find_package(prometheus-cpp CONFIG REQUIRED)
 endif()
 
-add_library(prometheus_exporter src/exporter.cc src/collector.cc
-                                src/exporter_utils.cc)
+add_library(opentelemetry_exporter_prometheus src/exporter.cc src/collector.cc
+                                              src/exporter_utils.cc)
 target_include_directories(
-  prometheus_exporter
+  opetelemetry_exporter_prometheus
```

Comment:
Typo.

opetelemetry_exporter_prometheus - > opentelemetry_exporter_prometheus

Reply:
Thanks. forgot to push all the changes after testing them locally. Have fixed now.

Reply:
Thanks, resolved then.